docs/asciidoc/static/configuration.asciidoc

```diff
@@ -358,7 +358,7 @@ What's an expression? Comparison tests, boolean logic, and so on!
 You can use the following comparison operators:
 
 * equality: ==,  !=,  <,  >,  <=,  >=
-* regexp: =~, !~
+* regexp: =\~, !~
```

Comment:
@peterskim12 there are two tildes there, do we need to escape both? WRT:
http://www.elasticsearch.org/guide/en/logstash/current/configuration.html#conditionals